cmd/src/cli/js-tests-scaffold/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "@holochain/holochain-nodejs": "^0.1.21",
+    "@holochain/holochain-nodejs": "^0.1.22",
```

Comment:
This seems like a problem.  Won't it break some of our tests cmd regression tests when building out a new feature?

Reply:
Not sure what you mean. 
I can't think of anything that this would break

Reply:
If we have a test, that tests whether the code generated by `hc init` works, i.e. runs `hc test` on that code, then it could fail because it would use a fixed version, not the "current" version in development.  At one point I thought we had such a test but maybe we don't.